src/main/java/com/minecolonies/coremod/permissions/ColonyPermissionEventHandler.java

```diff
@@ -195,8 +200,33 @@ public void on(final PlayerInteractEvent event)
                 }
             }
 
-            if (event.getItemStack().getItem() instanceof ItemMonsterPlacer
-                  && !colony.getPermissions().hasPermission(event.getEntityPlayer(), Permissions.Action.PLACE_HUTS))
+            @NotNull final EntityPlayer player = EntityUtils.getPlayerOfFakePlayer(event.getEntityPlayer(), event.getWorld());
+
+            if(event.getItemStack() != null
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') Change this condition so that it does not always evaluate to "true" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2583)